wen-gong/week5/album-app/app/models/album.rb

```diff
@@ -0,0 +1,4 @@
+class Album < ApplicationRecord
+
+    belongs_to :singer
```

Comment:
if you don't need albums to be make with the singer id you can add `optional: true` after `belongs_to :singer`